MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponent/PaymentComponentsController.swift

```diff
@@ -284,15 +284,15 @@ public final class PaymentComponentsController: PaymentComponentsProtocol {
     }
 
     public func obtainPDFURLFromPaymentRequest(paymentInfo: PaymentInfo, viewController: UIViewController) {
-        createPaymentRequest(paymentInfo: paymentInfo, completion: { [weak self] paymentRequestID, error in
+        createPaymentRequest(paymentInfo: paymentInfo, notifyDelegate: false, completion: { [weak self] paymentRequestID, error in
             if let paymentRequestID {
                 self?.loadPDFData(paymentRequestID: paymentRequestID, viewController: viewController)
             }
         })
     }
 
-    public func createPaymentRequest(paymentInfo: PaymentInfo, completion: @escaping (_ paymentRequestID: String?, _ error: GiniMerchantError?) -> Void) {
-        giniMerchant.createPaymentRequest(paymentInfo: paymentInfo) { result in
+    public func createPaymentRequest(paymentInfo: PaymentInfo, notifyDelegate: Bool = true, completion: @escaping (_ paymentRequestID: String?, _ error: GiniMerchantError?) -> Void) {
```

Comment:
@igor-gini could you please check how the issue was fixed in the `Health SDK` we shouldn't introduce an additional `notifyDelegate`